route/auth-router.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const {Router} = require('express')
+const Store = require('../model/store.js')
+const basicAuth = require('../lib/basic-auth-middleware.js')
+
+const authRouter = module.exports = new Router()
+
+authRouter.post('/admin/signup', (req, res, next) => {
+  Store.create(req.body)
+    .then(store => store.tokenCreate())
+    .then(token => {
+      res.cookie('X-SailPoint-Token', token, {maxAge: 604800000})
```

Comment:
great working setting a timeout on the cookies!

you should use the same timeout for your JWT tokens